test/test-support/CMakeLists.txt

```diff
@@ -3,7 +3,7 @@ project(TestSupportLibs)
 
 option(TEST_SOURCE_ROOT "src directory with test files")
 
-file(GLOB_RECURSE TEST_LIBS ${TEST_SOURCE_ROOT}/../*lib*.c)
+file(GLOB_RECURSE TEST_LIBS ${TEST_SOURCE_ROOT}/*lib*.c)
```

Comment:
This was looking in the repo root directory for test libraries. Likely a leftover from when we had a different repo structure. I've changed it to just look in the test directory.

This was causing issues after building the libclang bundle, as this was finding the llvm and clang source files as well.